gef.py

```diff
@@ -6374,18 +6374,18 @@ def do_invoke(self, _: List[str], **kwargs: Any) -> None:
         ctx = GlibcHeapWalkContext(min_size=args.min_size, max_size=args.max_size, count=args.count, summary=args.summary)
         if args.all or not args.arena_address:
             for arena in gef.heap.arenas:
-                self.dump_chunks_arena(arena, print_arena=args.all, allow_unaligned=args.allow_unaligned, ctx=ctx)
+                self.dump_chunks_arena(arena, ctx, print_arena=args.all, allow_unaligned=args.allow_unaligned)
```

Comment:
If we use the context idiom, I wonder whether we should put it as the first arg (which is what we do in Go). Do you know if it's also done this way in python?

Reply:
yea, i was hesitated on this one too - which name is better: context or options? eventually I chose context, because the remaining chunk count changes over time... options usually won't change. 

for the order of parameters, i don't really have strong opinion on that. what i usually do is to put parameters that absolutely required but need to be part of the workflow in the front (in this case - arena), then followed by the context (which contains mostly everything).

If you like the code to be modified, i can definitely do that by moving print_arena and allow_unaligned in - so it will be `dump_chunks_arena(arena, ctx)`, which is what i usually do.

Reply:
i have updated the code now. let's see how it looks.